ts/Series/AreaSpline/AreaSplineSeries.ts

```diff
@@ -143,12 +107,56 @@ SeriesRegistry.registerSeriesType('areaspline', AreaSplineSeries);
 
 /* *
  *
- *  Default export
+ *  Default Export
  *
  * */
 
 export default AreaSplineSeries;
 
+/* *
+ *
+ *  API Options
+ *
+ * */
+
+/**
+ * The area spline series is an area series where the graph between the
+ * points is smoothed into a spline.
+ *
+ * @sample {highcharts} highcharts/demo/areaspline/
+ *         Area spline chart
+ * @sample {highstock} stock/demo/areaspline/
+ *         Area spline chart
+ *
+ * @extends   plotOptions.area
+ * @excluding step, boostThreshold, boostBlending
+ * @product   highcharts highstock
+ * @apioption plotOptions.areaspline
+ */
+
+/**
+ * @see [fillColor](#plotOptions.areaspline.fillColor)
+ * @see [fillOpacity](#plotOptions.areaspline.fillOpacity)
+ *
+ * @apioption plotOptions.areaspline.color
+ */
+
+/**
+ * @see [color](#plotOptions.areaspline.color)
+ * @see [fillOpacity](#plotOptions.areaspline.fillOpacity)
+ *
+ * @apioption plotOptions.areaspline.fillColor
+ */
+
+/**
+ * @see [color](#plotOptions.areaspline.color)
+ * @see [fillColor](#plotOptions.areaspline.fillColor)
+ *
+ * @default   {highcharts} 0.75
+ * @default   {highstock} 0.75
```

Comment:
Does this have to be repeated? I thought `@default 0.75` would do?

Reply:
Good catch! It was all over the place.